### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by domthom21** [domthom21/eurocodedesign#6] [Ideas]
We should discuss and create our roadmap for the first features

**Comment by nc-hsu** [comment]
**Calculations that would be useful for Stahlbau I and II:**

- Section classification
- Section strength verification - plastic/elastic and with/without M-N-V interaktion
- Buckling + major and minor moment interaktion
- Calculate the geometic parameters of custom welded I and box sections
- Calculation of the elastic critical buckling moment for Lateral torsional buckling (biegedrillknicken)

**Other useful features could inlcude:**

- Printing out the calculation steps in a nice  way - to screen and also an option to print to PDF

**Comment by domthom21** [comment]
Add decorators for `@log`, like https://github.com/sighalt/logdecorator and do similar for `@stepper`

Stepper should support different languages (first en and de)

Also decorator `@NDP` for functions with Nationally Determined Parameters
